lib/charms/data_platform_libs/v0/data_interfaces.py

```diff
@@ -304,7 +303,7 @@ def _on_topic_requested(self, event: TopicRequestedEvent):
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 4
+LIBPATCH = 6
```

Comment:
Why not 5?

Reply:
This is the version of the library. I moved directly to version 6